package.json

```diff
@@ -16,6 +16,7 @@
     "eslint": "8.43.0",
     "eslint-config-next": "13.4.7",
     "next": "13.4.7",
+    "next-auth": "^4.22.1",
```

Comment:
Let's pin next-auth to a specific version

Reply:
fixed